tests/consensus_tests/test_triple_replication.py

```
@@ -73,30 +71,32 @@ def test_triple_replication(tmp_path: pathlib.Path):
+        for peer_api_uri in peer_api_uris:
+            count = get_collection_point_count(peer_api_uri, COLLECTION_NAME, exact=True)
+            print(count)
+
+        # Make sure there are no stale updates
+        upsert_random_points(peer_api_uris[0], 1, COLLECTION_NAME, offset=100_000, wait="true")
+
+        for peer_api_uri in peer_api_uris:
+            count = get_collection_point_count(peer_api_uri, COLLECTION_NAME, exact=True)
+            print(count)
+
+        assert False, f"Cluster did not converge: points_counts={points_counts}"
```

Ruff (0.15.10)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)